test/functorch/test_ops.py

```diff
@@ -774,6 +778,7 @@ def fn(inp, *args, **kwargs):
         skip("nn.functional.fractional_max_pool2d"),  # calls random op
         skip("nn.functional.fractional_max_pool3d"),  # calls random op
         xfail('nn.functional.scaled_dot_product_attention'),  # randomness
+        xfail('torch.ops.aten._efficient_attention_forward'),  # outputs ints
```

Comment:
for my curiosity: why does outputting ints cause breakage

Reply:
Ahh good question, if you grep for native functions that return integers as inputs in the code base, the only other one I can find is .item()

Alot of of our tests iterate over outputs calling methods that don't exist..

I think this gives more credence to this PR: https://github.com/pytorch/pytorch/pull/110546

That beings said I think we can still do that as followup